src/external/VTK.jl

```diff
@@ -0,0 +1,79 @@
+# Uses WriteVTK.jl to convert various structures and matrices to VTk file format
+using WriteVTK
+
+"""
+    EnergyToVTK(VTKFileName::String, energy::Energies{::Number})
+
+Stores the Energy structure in a VTK image file and returns the name of the file.
+"""
+function EnergyToVTK(VTKFileName::String, energy::Energies{<:Number})
+
+    vtkfile = vtk_grid(VTKFileName, 1, 1)
+    for i in keys(energy)
+        vtkfile[i] = energy.energies[i]
+    end
+    out = vtk_save(vtkfile)
+    return out[1]
```

Comment:
We don't use return if it's the last statement of a function.

Reply:
Yes I have noticed that convention in the package. Julia does automatically considers the last line as return value, but adding a return value makes the code more readable.

Reply:
I see that point, but IMHO that's debatable, especially in more functional languages like Julia, that often feature short functions. We decided to skip the returns, much inline with the Julia standard library.

Reply:
Cool, I will go with no return statements.